### PR TITLE
chore(deps): update all deps and fix database schema

### PR DESCRIPTION
**Pull request by ya-woodcutter**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "unity-cache",
-  "version": "2.1.0",
-  "description": "Cache abstraction around localforage.",
+  "version": "2.2.0",
```
Comment (evilj0e) [inline package.json:3]: You updated node to major version. I think it should been updated to major.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+  "description": "Cache abstraction around Dexie.",
```
Comment (evilj0e) [inline package.json:4]: 👍

File: src/index.js [hunk 3]
```diff
@@ -2,25 +2,23 @@ import Dexie from 'dexie';
...
-const EXPIRE_BIN = '___expire___';
-const EXPIRE_GLUE = '::';
 const DEFAULT_NAME = 'unity';
 const DEFAULT_VERSION = 1;
 
 const cacheInstance = {
-    config: {},
-    db: null
+    db: null,
+    config: {}
```
Comment (evilj0e) [inline src/index.js:10]: Useless change

File: src/index.js [hunk 4]
```diff
@@ -2,25 +2,23 @@ import Dexie from 'dexie';
...
-
+function initCacheConfig(stores = [], name = DEFAULT_NAME, version = DEFAULT_VERSION) {
+    stores = [].concat(stores);
     stores = stores.reduce((result, storeName) => {
         if (!RE_BIN.test(storeName)) {
             throw new UnityCacheError(`Store names can only be alphanumeric, '${storeName}' given`);
         }
 
-        result[storeName] = '&';
+        result[storeName] = '&key, value, expire';
```
Comment (evilj0e) [inline src/index.js:20]: What for?

File: src/index.js [hunk 5]
```diff
@@ -113,30 +133,23 @@ async function deleteDB() {
...
     }
 
-    return isValid ? await db[store].get(key) : null;
+    return isValid ? value : null;
 }
 
-async function set(store, key, value, expire = Number.MAX_SAFE_INTEGER) {
+async function set(store, key, value, ttl = Number.MAX_SAFE_INTEGER) {
     const { db } = cacheInstance;
+    const expire = Date.now() + Number(ttl);
```
Comment (evilj0e) [inline src/index.js:150]: Number('fdfff') // NaN

I think it would be better to check its type before

File: test/setup.js [hunk 6]
```diff
@@ -1,7 +1,9 @@
-require('browser-env')(['window', 'document', 'navigator']);
+require('@babel/polyfill');
+require('browser-env')([ 'window', 'document', 'navigator' ]);
 
 const indexedDB = require('fake-indexeddb');
 const FDBKeyRange = require('fake-indexeddb/lib/FDBKeyRange');
 
+window.Promise = global.Promise;
```
Comment (evilj0e) [inline test/setup.js:7]: Why?

**Review by evilj0e — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ya-woodcutter) [reply package.json:3]: Node is used only in testing environment. I'm not sure about the major version.

File: src/index.js (hunk 3 above)
Comment (ya-woodcutter) [reply src/index.js:10]: It's like a tree 🌲

File: test/setup.js (hunk 6 above)
Comment (ya-woodcutter) [reply test/setup.js:7]: Cause Dexie uses Promise from window.

File: src/index.js (hunk 4 above)
Comment (ya-woodcutter) [reply src/index.js:20]: To remove `EXPIRE_BIN`

File: src/index.js (hunk 5 above)
Comment (ya-woodcutter) [reply src/index.js:150]: I think it would be better to rewrite it to typescript. The check doesn't help at all.

**Review by evilj0e — APPROVED** [review]

**Pull request merged**